src/Amazon.SecretsManager.Extensions.Caching/Amazon.SecretsManager.Extensions.Caching.csproj

```diff
@@ -1,20 +1,32 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <CodeAnalysisRuleSet>..\..\code-analysis.ruleset</CodeAnalysisRuleSet>
-    <OutputType>Library</OutputType>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="AWSSDK.SecretsManager" Version="3.3.100.10" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.2.0" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.0.2">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
-  </ItemGroup>
-
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <CodeAnalysisRuleSet>..\..\code-analysis.ruleset</CodeAnalysisRuleSet>
+        <OutputType>Library</OutputType>
+        <!--        Addresses breaking change from .NET 5.0 to 6.0 in advance-->
+        <!--        https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/implicit-namespaces -->
+        <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
+        <PackageId>AWSSDK.SecretsManager.Caching</PackageId>
+        <PackageVersion>1.0.4</PackageVersion>
+        <Title>AWS Secrets Manager Caching for .NET</Title>
+        <Authors>Amazon Web Services</Authors>
+        <Description>The AWS Secrets Manager .NET caching client enables in-process caching of secrets for C# applications.</Description>
+        <Copyright>Amazon Web Services</Copyright>
+        <PackageProjectUrl>https://github.com/aws/aws-secretsmanager-caching-net</PackageProjectUrl>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
NuGet says we provide a URL to the Apache 2.0 license. This is not recommended, as the URL or the contents of the website it points to can change. For popular licenses, use `<PackageLicenseExpression>`